.github/workflows/ci-deploy.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Set up JDK 17
         uses: actions/setup-java@v4
         with:
-          distribution: temurin
+          distribution: liberica
```

Comment:
How does a different Java implementation impact the journal recovery output?

Reply:
the PR combines 2 PRs in essence; the change is in the recoverymanager

Reply:
Moved JDK change to PR - https://github.com/eXist-db/exist/pull/5480